### PR TITLE
Minor spelling and grammatical corrections in docs

### PR DESCRIPTION
**Pull request by JRodDynamite**
@intelsdi-x/snap-maintainers

### CODE REVIEW

File: docs/DISTRIBUTED_WORKFLOW_ARCHITECTURE.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ A distributed workflow is a workflow where one or more steps have a remote targe
...
+Distributed workflow is accomplished by allowing remote targets to be specified as part of a task workflow. This is done by having a gRPC server running that can handle actions needed by the scheduler to run a task. These are defined in the [managesMetrics](https://github.com/intelsdi-x/snap/blob/distributed-workflow/scheduler/scheduler.go) interface defined in scheduler/scheduler.go. This interface is implemented by both pluginControl in control/control.go and  ControlProxy in grpc/controlproxy/controlproxy.go. This allows the scheduler to not know/care where a step in the workflow is running. On task creation, the workflow is walked and the appropriate type is selected or created for each step in the workflow.
```
Comment (kindermoumoute) [inline docs/DISTRIBUTED_WORKFLOW_ARCHITECTURE.md]: >...and  ControlProxy...

typo: remove a space.

File: docs/BUILD_AND_TEST.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ Any `small` tests added to the Snap framework must conform to the following cons
 
 When complete, the full set of `small` tests for any given function or method should provide sufficient code coverage to ensure that any changes made to that function or method will not 'break the build'. This will assure the Snap maintainers that any pull requests that are made to modify or add to the framework can be safely merged (provided that there is sufficient code coverage and the associated tests pass).
 
-It should be noted here that the maintainers will refuse to merge any pull requests that trigger a failure of any of the `small` or `legacy` tests that cover the code being modified or added to the framework. As such, we highly recommend that contributors run the tests that cover their contributions locally before submitting their contribution as a pull request. Maintainers may also  ask that contributors add tests to their pull requests to ensure adequate code coverage before the they are willing to accept a given pull request, even if all existing tests pass. Our hope is that you, as a contributor, will understand the need for this requirement.
```
Comment (kindermoumoute) [inline docs/BUILD_AND_TEST.md]: > ...also  ask...

typo: remove a space

File: docs/DISTRIBUTED_WORKFLOW_ARCHITECTURE.md (hunk 1 above)
Comment (kindermoumoute) [inline docs/DISTRIBUTED_WORKFLOW_ARCHITECTURE.md]: > ...and  ControlProxy...

typo: remove space

File: docs/BUILD_AND_TEST.md (hunk 2 above)
Comment (kindermoumoute) [inline docs/BUILD_AND_TEST.md]: ...also  ask...
typo: remove a space

**Comment by mjbrender** [comment]
LGTM. Thanks @JRodDynamite

**Pull request merged**